xamarin/MainActivity.cs

```diff
@@ -62,7 +66,9 @@ private void ButtonStopServiceOnClick()
             SetButtonsEnabled(false);
 
             var intent = new Intent(Intent.ActionSend);
-            var componentName = new ComponentName("eu.fiskaltrust.androidlauncher", "eu.fiskaltrust.androidlauncher.Stop");
+            var componentName = FindViewById<RadioButton>(Resource.Id.radioGrpc).Checked
+                ? new ComponentName("eu.fiskaltrust.androidlauncher.grpc", "eu.fiskaltrust.androidlauncher.grpc.Stop")
```

Comment:
Should we use constants for these values?

Reply:
I'd rather leave this here so people can find it easier (most will only copy one part :) )